libdevcore/TrieHash.h

```diff
@@ -21,8 +21,9 @@
 
 #pragma once
 
-#include "Common.h"
-#include "FixedHash.h"
+#include <vector>
```

Comment:
`#include`s for stdlib should be after ones for our libraries, let's do it right it at least for new code